server/src/test/java/org/elasticsearch/ElasticsearchExceptionTests.java

```diff
@@ -505,12 +505,12 @@ public void testGetDetailedMessage() {
     public void testToXContent() throws IOException {
         {
             ElasticsearchException e = new ElasticsearchException("test");
-            assertExceptionAsJson(e, """
+            assertThrowableAsJson(e, """
```

Comment:
What's the reason for changing the assert method name from exception to throwable? Throwable will cover errors too, which we shouldn't be getting, i.e they terminate Elasticsearch.

Reply:
Sorry to be nitpicking here, it's really a minor point, but the argument of the method is an Exception, not a Throwable. If anything shouldn't both of them be called Exception?

Reply:
`generateThrowableXContent` takes a `Throwable`, so I can change the assert method to also take a `Throwable`, so everything matches.

Reply:
OK, yeah that seems the most consistent.